problems/secondSmallest.js

```diff
@@ -6,8 +6,22 @@
 * @returns {number} - Returns the second smallest number.
 */
 
-function secondSmallest() {
-  
+function secondSmallest(arr) {
+    let arr1 = 99;
+    let arr2 = 99;
```

Comment:
Let's give our variable names a bit more attention. They should often be nouns and always tell us what they are. So instead off arr1 and arr2 which would make you think they're arrays, let's name them smallest and secondSmallest. 

Also what would happen if the elements in the array were all over 100 ? This is why we shouldnt' start at 99. Instead we should start at Infinity. That way we can know for sure we'll only get smaller numbers.

Reply:
Understood